client/verta/verta/client.py

```diff
@@ -112,10 +112,12 @@ class Client(object):
         Currently active Project.
     expt : :class:`Experiment` or None
         Currently active Experiment.
+    _connect : str, default True
+        Whether to connect to server (``False`` for unit tests).
```

Comment:
Could you move this to the `Parameters` section? (line 99)

Reply:
fixed